fe/java-udf/pom.xml

```diff
@@ -125,7 +120,7 @@ under the License.
         </dependency>
         <dependency>
             <groupId>org.apache.hudi</groupId>
-            <artifactId>hudi-presto-bundle</artifactId>
+            <artifactId>hudi-hadoop-mr-bundle</artifactId>
```

Comment:
which one should be use? hadoop or presto?

Reply:
Both of two have the same implementation of `RecordReader` interface, and hadoop-bundle is smaller than presto-bundle.